client/blocks/inline-help/style.scss

```diff
@@ -0,0 +1,257 @@
+.sidebar .sidebar__footer .button,
```

Comment:
Just a quick thing, but styles that target inline-help as a child of sidebar should be added to the sidebar component, layout component, etc. Only direct `inline-help` styles should be on this file.